src/data/endpoints.ts

```diff
@@ -167,6 +168,20 @@ export const endpointsMap: EndpointsMap = {
       },
     },
   },
+  fee_stats: {
+    label: "Fee Stats",
+    endpoints: {
+      all: {
+        label: "Fee Stats",
```

Comment:
I think we should make this label "All Fee Stats" for consistency, what do you think?

Reply:
I think that's a good plan! I will update!

Reply:
I have changed the .all endpoint label to "All Fee Stats" for consistency.